resources/assets/sass/app.scss

```diff
@@ -69,6 +69,10 @@ legend {
     width: inherit;
 }
 
+.bold {
```

Comment:
If it's a utility class, we should probably prefix it with something like `u-`. I don't know if we have any other utility classes in Symposium we can use as reference points, though, but that's my general way of thinking. `.u-bold {}`